packages/bpk-theming/bpk-themeable-text/BpkThemeableText.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { cssModules } from 'bpk-react-utils';
+
+import STYLES from './bpk-themeable-text.scss';
+
+const getClassName = cssModules(STYLES);
+
+const BpkThemeableText = ({ children }) => {
+  const className = getClassName('bpk-themeable-text');
```

Comment:
Shouldn't we make BpkText themable instead of creating a bespoke component for this?

Reply:
I think it's better to do that in another PR to keep this one self-contained. Same approach we took with the native theming, have a 'proof-of-concept' component and then quickly migrate over to a 'real' component when it's ready.

Reply:
Fair enough

Reply:
will this file be removed once BpkText is themed?

Reply:
@matteo-hertel yep!

Reply:
awesome, just checking!

Reply:
Actually just to be clear - not necessarily `BpkText`, just any themed component we can use. Will probably be `BpkLink`.